core/src/main/java/novoda/lib/sqliteprovider/provider/SQLiteContentProviderImpl.java

```diff
@@ -96,14 +95,12 @@ private Uri insertSilently(Uri uri, ContentValues values) {
     protected int updateInTransaction(Uri uri, ContentValues values, String selection, String[] selectionArgs) {
         ContentValues insertValues = (values != null) ? new ContentValues(values) : new ContentValues();
```

Comment:
That means that the null guard is useless, as using `new ContentValues()` will also throw.